src/App.xaml.cs

```diff
@@ -46,7 +45,27 @@ public T GetService<T>()
 
     public static WindowEx MainWindow { get; } = new MainWindow();
 
-    internal static NavConfig NavConfig { get; } = System.Text.Json.JsonSerializer.Deserialize(File.ReadAllText(Path.Combine(AppContext.BaseDirectory, "navConfig.json")), SourceGenerationContext.Default.NavConfig)!;
+    private static string RemoveComments(string text)

```

Comment:
Why are we adding commented out items to navconfig just to remove them here?  Would it be better to add an optional "experimental id" field to navconfig and then update the nav menu generation code to check that field against local settings?

Reply:
the latter is done in this PR already - the comments are so that we can have comments that explain how to add an experimental feature.